package.json

```diff
@@ -111,6 +111,7 @@
     "eslint-plugin-jest": "~26.5.3",
     "eslint-plugin-prettier": "~4.0.0",
     "eslint-plugin-react": "~7.29.4",
+    "hatchling": "^0.2.2",
```

Comment:
@athornton Why is this additional dependency needed?

Reply:
Good question.  Let's take it out and see if it still works.  Probably a result of trying to run test suites locally and getting confused?